projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/components/case-file-view-folder/case-file-view-folder.component.spec.ts

```diff
@@ -34,26 +36,50 @@ describe('CaseFileViewFolderComponent', () => {
       document_filename: 'Lager encyclopedia',
       document_binary_url: '/test/binary',
       attribute_path: '',
-      upload_timestamp: '11 May 2023 00:00:00'
+      upload_timestamp: '2023-05-11T11:15:10.00'
```

Comment:
Have you checked that this is the format returned by the DocumentsAndCategories API?

Reply:
Yes @andywilkinshmcts